.gitpod.yml

```diff
@@ -0,0 +1,15 @@
+image:
+  file: .gitpod.Dockerfile
+
+ports:
+  # Sidecar API port
+  - port: 3999
```

Comment:
I'm getting an error when I access code node endpoints (e.g. `/v2/info`). it says "Port 3999 didn't respond"... Any idea why?

Reply:
Should be fixed now. Too many environmental variables 😅